util/test/check-test-venv.bash

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+if [ -z "$CHPL_HOME" ]; then
+  # compute the chpl home directory
+  export CHPL_HOME=$(cd $(dirname $0) ; cd ..; cd ..; pwd)
+fi
+
+python=$($CHPL_HOME/util/config/find-python.sh)
+venv_path=$("$python" "$CHPL_HOME/util/chplenv/chpl_home_utils.py" --venv)
+
+if [ -f "$venv_path/chpl-test-reqs" ]
```

Comment:
Note that `CHPL_TEST_VENV_DIR=none` / `CHPL_TEST_VENV_DIR=<dir>` have historically allowed users to either provide the dependencies in their main environment, or specify a directory that contains the venv. I use the former to avoid having to build the venv on my laptop, and the latter to just do a single build on cray's instead of building for each new system I use. See the old `util/test/activate_chpl_test_venv.py` for more info.

I think this check breaks those 2 workflows, but am not positive.

Reply:
No problem I can fix. I might ask you to help test it though.

Reply:
@ronawho - OK, thanks. I have updated. Please have a look and try it out in a `CHPL_TEST_VENV_DIR` setting.